code/game/gamemodes/blob/blobs/blob_mobs.dm

```diff
@@ -229,13 +229,29 @@
 
 /mob/living/simple_animal/hostile/blob/blobbernaut/Life()
 	if(..())
+		var/list/blobs_in_area = range(2, src)
 		if(independent)
 			return // strong independent blobbernaut that don't need no blob
 		var/damagesources = 0
-		if(!(locate(/obj/structure/blob) in range(2, src)))
+		if(!(locate(/obj/structure/blob) in blobs_in_area))
 			damagesources++
 		if(!factory)
 			damagesources++
+		else
+			if(factory && (locate(/obj/structure/blob/core) in blobs_in_area))
```

Comment:
the point of the else is to remove the factory checks

Reply:
ohh